### ISSUE TITLE
Invalid statistic_id

### ISSUE THREAD

**Issue by pgerner** [ZsBT/hass-w1000-portal#13]
Hi Zsombor,

I have entries in my logfile every time I restart HA for each sensor (report in EON) configured:

2023-01-13 21:11:18.084 WARNING (MainThread) [custom_components.w1000-energy-monitor] exception at async_import_statistics 'sensor.w1000_Consumption': Invalid statistic_id
2023-01-13 21:11:18.258 WARNING (MainThread) [custom_components.w1000-energy-monitor] exception at async_import_statistics 'sensor.w1000_Production': Invalid statistic_id

I have values for the sensors and even the consumption is showing, although as a big peak instead of hourly values:
![image](https://user-images.githubusercontent.com/122386217/212411616-5cc4d341-aeb5-45fc-aca8-70c74164e2b1.png)

Could you help me to troubleshoot?

Thanks
Peter

**Comment by lazlykama** [comment]
Same here, something brake at 13th Jan in the Energy dashboard. I see similar view, but with bigger spike. 

![image](https://user-images.githubusercontent.com/118392651/212537214-333c7bda-73e8-4109-8205-3584fde54086.png)

**Comment by ZsBT** [comment]
Hi, it seems for the 13th, there was a lack in EON data providing. I had holes, too.
What happens if you take a look at that day now?

**Comment by pgerner** [comment]
No change for me, seeing the same spike both for today and the past days . I don’t think it’s related

**Comment by pgerner** [comment]
And the invalid statistics Id is still reporting in the logs. Checking the code it appears when statistic_id is not the proper format (delimited by :). Can’t it be the root cause?

**Comment by ZsBT** [comment]
Nice catch! Can you copy/paste the configuration entry (of course without username/password), and if possible, a detailed log as described on the wiki https://github.com/ZsBT/hass-w1000-portal/wiki/Troubleshooting ?

**Comment by lazlykama** [comment]
Just for some extra info: I don't have this error in the log, while after 13th and 14th as days with failed read (without valid, hourly information) yesterday (15th) looks normal and the module working perfectly.

![image](https://user-images.githubusercontent.com/118392651/212762473-b7a478b3-1102-41a2-8d04-bc032584e358.png)

**Comment by pgerner** [comment]
Yep, you found the root cause. After renaming the report names to lowercase on EON and modifying the config accordingly, there are no errors in the log and yesterday's consumption is showing properly. Thanks a lot for your help Zsombor!

**Comment by ZsBT** [comment]
Thanks for the debugging. I'll fix this in the upcoming days!

**Comment by Gollam** [comment]
@ZsBT any progress hopefully?

**Comment by ZsBT** [comment]
could not have a chance to fix this... but I'm still on it!

**Comment by lupus78** [comment]
Thanks, your module works nicely. 
Can you at least update the README to include that the profile names have to be lowercase. I ran into this issue too.

**Comment by zee36** [comment]
Hi Everyone,

Did you guys notice any errors of reading the DP_1-1:1.8.00 or DP_1-1:2.8.00 curves? These are unavailable for me since 29 Nov, while +A and -A curves are reading data... Maybe some bugs at EON's side, just wanted to check if this is the case for anyone else.

Thanks a lot.

See below my logs:

2023-12-06 19:54:20.829 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report ha.import
2023-12-06 19:54:24.261 DEBUG (MainThread) [custom_components.w1000-energy-monitor] resp status http 200
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report ha.import in workarea Munkaterület1702
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report export in workarea Munkaterület1702
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report ha.export in workarea Munkaterület1702
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report import in workarea Munkaterület1702
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report griddown in workarea Munkaterület1290
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report griddown in workarea Munkaterület1290
2023-12-06 19:54:24.423 DEBUG (MainThread) [custom_components.w1000-energy-monitor] found report gridup in workarea Munkaterület1290
2023-12-06 19:54:24.952 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report ha.export
2023-12-06 19:54:25.161 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report import
2023-12-06 19:54:25.324 DEBUG (MainThread) [custom_components.w1000-energy-monitor] import statistic: sensor.w1000_import count: 42
2023-12-06 19:54:25.324 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report export
2023-12-06 19:54:25.443 DEBUG (MainThread) [custom_components.w1000-energy-monitor] import statistic: sensor.w1000_export count: 42
2023-12-06 19:54:25.444 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report griddown
2023-12-06 19:54:25.538 DEBUG (MainThread) [custom_components.w1000-energy-monitor] import statistic: sensor.w1000_griddown count: 42
2023-12-06 19:54:25.539 DEBUG (MainThread) [custom_components.w1000-energy-monitor] reading report gridup
2023-12-06 19:54:25.632 DEBUG (MainThread) [custom_components.w1000-energy-monitor] import statistic: sensor.w1000_gridup count: 42
2023-12-06 19:54:25.633 DEBUG (MainThread) [custom_components.w1000-energy-monitor] Notifying all listeners
2023-12-06 19:54:26.014 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_ha_import
2023-12-06 19:54:26.015 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_ha_export
2023-12-06 19:54:26.015 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_import
2023-12-06 19:54:26.016 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_export
2023-12-06 19:54:26.017 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_griddown
2023-12-06 19:54:26.018 DEBUG (MainThread) [custom_components.w1000-energy-monitor] registered sensor: sensor.w1000_gridup
2023-12-06 19:54:26.025 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': None, 'unit': 'kWh', 'attributes': {'curve': 'Home DP_1-1:1.8.00', 'generated': None, 'state_class': 'total_increasing', 'device_class': 'energy'}}
2023-12-06 19:54:26.027 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': None, 'unit': 'kWh', 'attributes': {'curve': 'Home_6 DP_1-1:2.8.00', 'generated': None, 'state_class': 'total_increasing', 'device_class': 'energy'}}
2023-12-06 19:54:26.028 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': 2.005, 'unit': 'kWh', 'attributes': {'curve': 'Home +A', 'generated': '2023-12-06T18:00:00', 'state_class': 'total', 'device_class': 'energy'}}
2023-12-06 19:54:26.029 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': 0.0, 'unit': 'kWh', 'attributes': {'curve': 'Home -A', 'generated': '2023-12-06T18:00:00', 'state_class': 'total', 'device_class': 'energy'}}
2023-12-06 19:54:26.030 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': None, 'unit': 'kWh', 'attributes': {'curve': 'Home DP_1-1:1.8.00', 'generated': None, 'state_class': 'total_increasing', 'device_class': 'energy'}}
2023-12-06 19:54:26.030 DEBUG (MainThread) [custom_components.w1000-energy-monitor.sensor] {'state': None, 'unit': 'kWh', 'attributes': {'curve': 'Home DP_1-1:2.8.00', 'generated': None, 'state_class': 'total_increasing', 'device_class': 'energy'}}

**Comment by vakesz** [comment]
EON has sent out a mail few days ago that they're having issues, probably it's the root cause of this behaviour

**Comment by vakesz** [comment]
![kép](https://github.com/ZsBT/hass-w1000-portal/assets/3627611/962da970-bd03-450d-867a-8a7efe1a1c5a)

**Comment by zee36** [comment]
Cheers vakesz, hope this is the root cause

**Comment by vakesz** [comment]
Database information seem to be available again from Távleolvasási Portal.
